ledger/participant-integration-api/src/main/resources/db/migration/postgres/V50__Append_only_schema.sql

```diff
@@ -121,52 +121,541 @@ CREATE TABLE participant_events_divulgence             PARTITION OF participant_
 CREATE TABLE participant_events_create                 PARTITION OF participant_events FOR VALUES IN (10);
 CREATE TABLE participant_events_consuming_exercise     PARTITION OF participant_events FOR VALUES IN (20);
 CREATE TABLE participant_events_non_consuming_exercise PARTITION OF participant_events FOR VALUES IN (25);
+*/
+
+---------------------------------------------------------------------------------------------------
+-- Events table: divulgence
+---------------------------------------------------------------------------------------------------
+CREATE TABLE participant_events_divulgence (
+    -- * kinds of events
+    event_kind smallint NOT NULL CONSTRAINT divulgence_event_kind CHECK (event_kind = 0),
```

Comment:
Why not add the kind in the view directly as constants?

Reply:
👍